app/boards/shields/launchpad/Kconfig.defconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_LAUNCHPAD
+
+config ZMK_KEYBOARD_NAME
+    default "LaunchPad"
+
+config ZMK_USB
```

Comment:
Is there a specific reason you explicitly enabled USB here? That's usually handled by the board instead of the shield.